test_conformance/math_brute_force/FunctionList.h

```diff
@@ -88,6 +88,7 @@ typedef struct Func
   int             ftz;
   int             relaxed;
   const vtbl      *vtbl_ptr;
+  float float_embedded_relaxed_ulps;
```

Comment:
Somewhat of a cosmetic comment, but I would find these changes easier to understand if this field was listed in the structure after `relaxed_error`, and if it were renamed slightly - maybe `relaxed_embedded_error`?  This would make it more obvious that these two values are related.

Reply:
No problem in renaming, but I'm a bit reluctant to re-order.

Doing so would also involve changing the macros in `FunctionList.cpp`. I've managed to avoid that thus far as the new member is at the end of the struct, so does not break the implicit ordering of the existing macros.

Reply:
Would this though?  I agree we'd need to modify `ENTRY_EXT`, but we're doing this anyways.  I don't think we'd need to modify any of the others though, we'd just need to insert `INFINITY` into the middle of the structure vs. the end.

Reply:
It was the addition of `INFINITY` that I was getting at as I usually prefer to limit the scope of changes for bug fixes.

Regardless, I've made the change now.